### PR TITLE
[12.0][IMP] hr_employee_service: support _get_date_start_work

### PR DESCRIPTION
**Pull request by alexey-pelykh**
To support https://github.com/odoo/odoo/commit/90731ad170c503cdfe89a9998fa1d1e2a5035c86

### CODE REVIEW

**Review by jarroyomorales — APPROVED** [review]
Nice, I dont like the odoo's function since the date an employee is created and the real service start day arent always the same and this solves it.

File: hr_employee_service/models/hr_employee.py [hunk 1]
```diff
@@ -107,3 +107,7 @@ def _compute_service_duration_display(self):
     def _onchange_service_hire_date(self):
         if not self.service_start_date:
             self.service_start_date = self.service_hire_date
+
+    # NOTE: Support odoo/odoo@90731ad170c503cdfe89a9998fa1d1e2a5035c86
+    def _get_date_start_work(self):  # pragma: no cover
```
Comment (pedrobaeza) [inline hr_employee_service/models/hr_employee.py]: Don't trick coverage on this

Comment (alexey-pelykh) [reply hr_employee_service/models/hr_employee.py]: Removed

**Review by pedrobaeza — APPROVED** [review]

**Pull request closed without merging**